### PR TITLE
Storybook: Stellar Send Note/Memo

### PR DESCRIPTION
**Pull request by nathunsmitty**
Story📕: [Desktop](http://localhost:6006/?selectedKind=Wallets%2FSendForm%2FNote%20and%20Memo&selectedStory=Normal%20input&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel), [Mobile](http://localhost:7007/?selectedKind=Wallets%2FSendForm%2FNote%20and%20Memo&selectedStory=Normal%20input&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

I ended up combining the `Note` and `Memo` components into one component, as the two shared a fair amount of styling and combining them didn't really add a ton of complexity.

Note that this PR also adds a `transformText` method to `PlainInput`. I needed some way to insert emojis into the input field. I know @buoyad mentioned there were some issues with it, though, so please let me know if you have a better solution to the emoji insertion problem.

As I've started to learn (and am very unsurprised by), [unicode in JavaScript is kinda weird](https://mathiasbynens.be/notes/javascript-unicode). Since there are emojis and user input in this PR, I'd really appreciate a look over that part just in case there is any unicode havoc one could wreck.

### CODE REVIEW

**Comment by nathunsmitty** [comment]
ready 4 review @keybase/react-hackers!

**Comment by nathunsmitty** [comment]
@keybase/react-hackers bump!

**Comment by cjb** [comment]
@buoyad, please could you take a look?  Thanks!

File: shared/wallets/send-form/note-and-memo/index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+          {!!this.props.memoError && (
+            <Kb.Text type="Body" style={styles.errorMessage}>
+              {this.props.memoError}
+            </Kb.Text>
+          )}
+        </Kb.Box2>
+        <Kb.Divider
+          style={Styles.collapseStyles([
+            styles.divider,
+            this.props.noteError ? {backgroundColor: Styles.globalColors.red} : {},
```
Comment (buoyad) [inline shared/wallets/send-form/note-and-memo/index.js]: I think `noteError` -> `memoError`

**Pull request merged**